client.go

```diff
@@ -33,6 +33,7 @@ func RunClient(addr string, uploadBytes, downloadBytes uint64) error {
 	}
 	log.Printf("uploaded %s: %.2fs (%s/s)", formatBytes(uploadBytes), uploadTook.Seconds(), formatBytes(bandwidth(uploadBytes, uploadTook)))
 	log.Printf("downloaded %s: %.2fs (%s/s)", formatBytes(downloadBytes), downloadTook.Seconds(), formatBytes(bandwidth(downloadBytes, downloadTook)))
+	fmt.Printf("{ latencies: [ %f, %f ]}\n", uploadTook.Seconds(), downloadTook.Seconds())
```

Comment:
```suggestion
	fmt.Printf("{ latencies: [ %f ]}\n", uploadTook.Seconds() + downloadTook.Seconds())
```

Sorry. Bad specification. We only care about the entire latency (for now). We have one benchmark only uploading and one only downloading. Can be extended in future versions.